src/spec/music_album_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../Classes/music_album'
+
+describe MusicAlbum do
+  context 'Unit test for Music Album' do
```

Comment:
Please add a before each block after describe to initialize a music object for each test